modules/setting/setting.go

```diff
@@ -471,6 +472,22 @@ func IsRunUserMatchCurrentUser(runUser string) (string, bool) {
 	return currentUser, runUser == currentUser
 }
 
+func createPIDFile(pidPath string) {
+	currentPid := os.Getpid()
+	if err := os.MkdirAll(filepath.Dir(pidPath), os.ModePerm); err != nil {
+		log.Fatal(4, "Can' create PID folder on %s", err)
+	}
+
+	file, err := os.Create(pidPath)
```

Comment:
Quoting bkcsoft on this one:

> Overall, change the format to this instead whenever possible. Keeps errors(variables) bound to their context(block).

`if err := os.Create(...); err != nil {
  [...]
}`